src/main/resources/camel/odoo-patient-handler.xml

```diff
@@ -23,7 +23,7 @@
 
         <choice>
             <when>
-                <simple>${exchangeProperty.patient-odoo-id} == null &amp;&amp; ${exchangeProperty.patient.get('patientVoided')} == true</simple>
+                <simple>${exchangeProperty.patient-odoo-id} == null &amp;&amp; ${exchangeProperty.patient.get('voided')} == true</simple>
```

Comment:
I think you might want to check both `voided` and `patientVoided`, a person can be linked to more than one entity e.g. User, Provider and Patient. Therefore, a patient account can get voided but not necessarily their person record, or user or provider accounts.

Reply:
@wluyima it seems like we have `voided` attribute in `patient` model but AFAIK we don't have any attribute as `patientVoided`
We have `personVoided` in `person` model.
CMIIW

Reply:
Hello @wluyima,
I am not complete sure how this data structure works, but here it seems that it's getting the `patient` info and not the `person`, so that should not happen. Or are you saying this `patient` is actually a `person` but if that's the case it should not know if the `patient` is voided or not, since as you pointed it may not be a `patient` at all.

Reply:
Sorry! That's what I meant to check both Person.personVoided and Patient.voided

Reply:
@VaishSiddharth  @wluyima  why doing test on both properties personVoided and Patient.voided  as in this route we deal with patient data only ( and with patient identifier,...) and only patient are synchronized with Odoo.


IMHO, The use case "a patient is voided but the person is not voided" should be ignored as well  as this route need patients only. If a paitent is voided, we will have an error when trying to get the patient identifier and it causes erros in this route when a patient is voided.

Why should the use case "a patient is voided but the person is not voided" be processed  in this route ?

Reply:
If the route only processes patients, then it's all good.